lib/network_utils.ml

```diff
@@ -705,9 +705,16 @@ end
 
 module Ovs = struct
 
-	module Cli = struct
+	module Cli : sig
+		val vsctl : ?log:bool -> string list -> string
+		val ofctl : ?log:bool -> string list -> string
+		val appctl : ?log:bool -> string list -> string
+	end = struct
+	let s = Semaphore.create 5
```

Comment:
I think the module (`struct`..`end`) should be indented just like the signature. What is `ocp-indent` saying?